app/controllers/concerto_plugins_controller.rb

```diff
@@ -106,14 +106,22 @@ def destroy
   end
 
   def update_gem
+    results = {}
     plugin = ConcertoPlugin.find(params[:id])
-    system('bundle update', plugin.gem_name)
-    rake_precompile()
-    restarted = restart_webserver()
-    if restarted
-      flash[:notice] = t(:plugin_updated)
+    results[:bundle_output] = `bundle update #{plugin.gem_name}`
```

Comment:
@mfrederickson That command is a bit funny - it tends not to do what you expect: http://stackoverflow.com/questions/11263150/update-just-one-gem-with-bundler . We should use bundle update --source gem-name